cmd/gearlogger/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/Clever/gearadmin"
+	"gopkg.in/Clever/kayvee-go.v1"
+)
+
+func logMetrics(g gearadmin.GearmanAdmin) {
+	statuses, err := g.Status()
+	if err != nil {
+		log.Fatalf("error retrieving gearman status: %s", err)
+	}
+	for _, status := range statuses {
+		fmt.Println(kayvee.FormatLog("gearlogger", "info", "status", map[string]interface{}{
+			"function": status.Function,
+			"running":  status.Running,
+			"total":    status.Total,
+			"workers":  status.AvailableWorkers,
+		}))
```

Comment:
Yeah stdout is fine, especially since this can also just be run as a command line tool.  Usually we suggest stderr for metrics from cli tools if the metrics themselves are not the actual output (so that the actual output can still be used for piping between cli tools), but thats not the case here.  Within the docker container we should be capturing both.